1.0/nanoserver/sdk/projectjson/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM microsoft/nanoserver:10.0.14393.447
+FROM microsoft/nanoserver:10.0.14393.575
 
 # Install .NET Core SDK
-ENV DOTNET_SDK_VERSION 1.0.0-preview2-003154
+ENV DOTNET_SDK_VERSION 1.0.0-preview2-003158
```

Comment:
This change is confusing?  Did you rebase your changes prior to making the PR?  This shouldn't be showing up as a change.